platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/widgets/MyLocationView.java

```diff
@@ -58,6 +58,7 @@
 
   private LatLng latLng;
   private Location location;
+  private LocationEngine locationSource;
```

Comment:
I'd rather call it `locationEngine` even if `LocationSource` is our default implementation.